python/cudf/cudf/io/json.py

```diff
@@ -189,19 +189,27 @@ def to_json(
 ):
     """{docstring}"""
 
+    if engine == "auto":
+        engine = "pandas"
+
     if engine == "cudf":
-        return_as_string = False
+        orient = kwargs.pop("orient", None)
```

Comment:
```suggestion
```
Can we just introduce this as a function parameter instead of extracting from `kwargs`, that makes getting rid of `kwargs` easier in future.

Reply:
added `orient`.